src/sentryFilter/index.js

```diff
@@ -0,0 +1,58 @@
+/**
+ * Copyright 2020 Inrupt Inc.
+ *
+ * Permission is hereby granted, free of charge, to any person obtaining a copy
+ * of this software and associated documentation files (the "Software"), to deal in
+ * the Software without restriction, including without limitation the rights to use,
+ * copy, modify, merge, publish, distribute, sublicense, and/or sell copies of the
+ * Software, and to permit persons to whom the Software is furnished to do so,
+ * subject to the following conditions:
+ *
+ * The above copyright notice and this permission notice shall be included in
+ * all copies or substantial portions of the Software.
+ *
+ * THE SOFTWARE IS PROVIDED "AS IS", WITHOUT WARRANTY OF ANY KIND, EXPRESS OR IMPLIED,
+ * INCLUDING BUT NOT LIMITED TO THE WARRANTIES OF MERCHANTABILITY, FITNESS FOR A
+ * PARTICULAR PURPOSE AND NONINFRINGEMENT. IN NO EVENT SHALL THE AUTHORS OR COPYRIGHT
+ * HOLDERS BE LIABLE FOR ANY CLAIM, DAMAGES OR OTHER LIABILITY, WHETHER IN AN ACTION
+ * OF CONTRACT, TORT OR OTHERWISE, ARISING FROM, OUT OF OR IN CONNECTION WITH THE
+ * SOFTWARE OR THE USE OR OTHER DEALINGS IN THE SOFTWARE.
+ */
+
+import { partition } from "../enumerable";
+
+function hasBreadcrumbs(event) {
+  return !!(event && event?.breadcrumbs && event?.breadcrumbs?.length);
+}
+
+export function containsSensitiveInformation(string) {
+  return !!string.match(/code=.+\b/);
```

Comment:
You can do away with this regex here
```suggestion
  return string.contains("code=");
```

The regex may also potentially have unbounded complexity:
<img width="716" alt="Screenshot 2022-03-25 at 14 50 03" src="https://user-images.githubusercontent.com/30827/160133263-677801a6-8e9a-4edb-985e-2967b996ea58.png">

Reply:
(looked at that here: http://xenon.stanford.edu/~xusch/regexp/ )

Reply:
I _think_ you mean `includes` here but you're right. I forgot about `String.includes`.